en/security.md

```diff
@@ -0,0 +1,20 @@
+# Security  
+
+There are two types of APIs: those that have security criteria, and those that do not.
+
+An API is considered to have security criteria if the answer to at least one of the following 3 questions is “yes”:
+  1. _Visibility_: Is the API exposed to outside of Vaudoise ?
+  2. _Data Type_: Does the API allow access to restricted (e.g. personal data, ...) or confidential data (e.g. personal, sensitive data, ...) according to Vaudoise confidentiality definition levels ?
+  3. _Data Access_: Does the API allow data creation, modification or deletion ?
+
+These questions and their answers *MUST* be included in the documentation for each API (see General Documentation chapter). In addition, the column “Security Case” (internet, sensitive data, data update) in the document headers *MUST* be filled.
+
+If an API has security criteria, then the following requirements *MUST* be met.
+ 
+  - Integration into the API manager (mandatory if the API is exposed to the Internet, recommended otherwise)   
+  - Rights managed up to the method level and not just at the API level, And the access model to be documented  (IAM sheet) (covered by the API Manager)   
+  - Authenticated API calls using tokens (covered by API Manager)   
+  - Logging – API calls must be logged and kept for 90 days (covered by the API Manager, with checks for log management and service logs)   
```

Comment:
Not sure if I understood it correctly. But do the services also have to keep the application /access logs for 90Days? Or is it only the API manager logs?
 I Believe we don't have a long retention period on our side.

Reply:
Good question, I'll ask our security colleagues to answer.

Reply:
@hugo-ma-alves : It's in french (sorry :-/), but here is the answer.

If there are logs at the API Manager level, then it's perfect for our security colleagues. 

![image](https://user-images.githubusercontent.com/1738499/201312694-35e750d5-10a6-477d-8c7e-90a1a23639b9.png)